### PR TITLE
Convert all Node tests to Model tests

### PR DESCRIPTION
**Pull request by bddppq**
https://github.com/onnx/onnx/issues/590

### CODE REVIEW

**Comment by bddppq** [comment]
this is based on https://github.com/onnx/onnx/pull/650

**Comment by bddppq** [comment]
this needs https://github.com/onnx/onnx/pull/652

**Comment by bddppq** [comment]
@houseroad Do you want to wait for other open PRs that are related to backend tests to be merged first?

**Comment by houseroad** [comment]
@bddppq I think it's ok to merge this first. I will review later... Let me first unblock you :-)

**Comment by bddppq** [comment]
@houseroad This one is not blocking me, feel free to make your choice :-)

File: onnx/backend/test/case/test_case.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import namedtuple
+
+TestCase = namedtuple('TestCase', [
```
Comment (houseroad) [inline onnx/backend/test/case/test_case.py]: Probably add another two fields: atol, rtol for future use?

Also some comments to explain what the purpose of each field is?

Comment (bddppq) [reply onnx/backend/test/case/test_case.py]: atol and rtol should be added in this PR https://github.com/onnx/onnx/pull/628

**Pull request merged**